### DISCUSSION TITLE
Why are all the browsers detecting virus in v0.91?

### DISCUSSION THREAD

**Discussion by Mionwang** [seerge/g-helper#689] [Q&A] [answered]
The 0.91 update won't install nor will any browser let me download it. It says "Failed- Virus detected"

**Comment by seerge** [comment, marked as answer]
@Mionwang hello

False positives from Windows Defender (or any other similar system that uses machine learning for detection) is possible as application is not digitally signed with a certificate. You can always download a version below or compile app by yourself. All application sources are open and can be monitored from A to Z :)

https://github.com/seerge/g-helper#windows-defender-marks-app-download-as-malware--virus

**Reply by Mionwang** [reply]
thanks :)
